pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
-name = "heal"
-version = "0.1.2"
+name = "heal-sdk"
+version = "0.1.3"
```

Comment:
feels like we can announce this as release `1.0.0` since the VLMD stuff is pretty significant

Reply:
Yes!!